xbmc/filesystem/CurlFile.h

```diff
@@ -209,5 +211,6 @@ namespace XFILE
       MAPHTTPHEADERS m_requestheaders;
 
       long            m_httpresponse;
+      bool            m_isshoutcast;
```

Comment:
curl as a generic class should not know that something like shoutcast does exist.